jest.test.config.js

```diff
@@ -12,5 +12,5 @@ module.exports = {
     // Transpile also our local packages as they are only symlinked.
     'node_modules/(?!(@commercetools-[frontend|backend]+)/)',
   ],
-  testEnvironment: 'jest-environment-jsdom-sixteen',
+  testEnvironment: 'jsdom',
```

Comment:
jsdom "standard" now uses v16 of jsdom. The sixteen variant is not really needed anymore. I suggest however we keep the `jest-present-for-jsdom-sixteen` with the `jest-environment-jsdom-sixteen` as it otherwise would be breaking. We could there remove the `jest-environment-jsdom-sixteen` to `jsdom` but that could also be another PR at some point or just when it's removed.